automation/roles/pgbouncer/tasks/main.yml

```diff
@@ -36,6 +36,14 @@
     mode: "0750"
   tags: pgbouncer_conf, pgbouncer
 
+- name: Fetch PostgreSQL TLS certificate, key and CA from the master node
+  ansible.builtin.include_role:
+    name: ../roles/tls_certificate/copy
+  vars:
+    copy_for: "pg"
+  when: tls_cert_generate|bool
+  tags: pgbouncer_conf, pgbouncer
```

Comment:
I think we could call this role at the `deploy_pgcluster.yml` playbook level rather than from the tasks file.

```
   - role: tls_certificate/copy
      vars:
        copy_for: "pg"
```

Reply:
I will have to check if the same thing might apply as with etcd suggestion.

Reply:
The will now be called from `deploy_pgcluster.yml`